flocker/node/agents/pd.py

```diff
@@ -0,0 +1,383 @@
+# -*- test-case-name: flocker.node.agents.functional.test_pd -*-
+# Copyright ClusterHQ Inc.  See LICENSE file for details.
+
+"""
+A PD implementation of the ``IBlockDeviceAPI``.
```

Comment:
1. Please define "PD".
2. Link to relevant documentation (API and tutorial) to ease life for reviewers and later maintainers.